lib/osf-components/addon/components/tags-widget/template.hbs

```diff
@@ -1,4 +1,5 @@
-{{#tag-input
+{{! template-lint-disable }}
+{{#tag-input 
     data-test-tags-widget-tag-input
     (html-attributes aria-label=(t 'osf-components.tags-widget.tags'))
```

Comment:
Note: because of this line, I can't convert this to an angle-bracket invocation, and the data-test selector prevents it from linting properly. So I had to disable template-lint for this component invocation.

Reply:
Should be able to do:
```hbs
<TagInput
    data-test-tags-widget-tag-input
    aria-label={{t 'osf-components.tags-widget.tags'}}
    .
    .
    .
```

Reply:
Perfect, thank you.